ouroboros-network/src/Ouroboros/Network/Socket.hs

```diff
@@ -101,12 +102,15 @@ socketAsMuxBearer sd = do
       closeSocket = Socket.close sd
 
       sduSize :: IO Word16
+#if defined(mingw32_HOST_OS)
+      sduSize = return 13000 -- MaxSegment isn't supported on Windows
```

Comment:
How did we pick this value?

Reply:
I have a vague memory that you and Neil wanted 13000 but I'm unable to find any trace of it on slack.
Anyway it should be large but smaller than the initial window size (10*mss).

Reply:
It would be good to place this in the comment.